docs/synapse/deploymentguide.rst

```diff
@@ -354,6 +356,38 @@ Start the container::
     If you are deploying a mirror from an existing large Cortex, this startup may take a while to complete
     initialization.
 
+.. _deployment-guide-storm-pool:
+
+Configure a Storm Query Pool (optional)
+=======================================
+
+A ``Cortex`` may be configured to use a pool of mirrors in order to offload ``Storm`` query execution and distribute
```

Comment:
We don't use inline literals for other references to the word Cortex in this document.

A person following this guide has not yet been introduced to the storm tool, so someone following along with this guide step by step cannot complete it. We'd need to add a example to start the storm CLI tool directly in the container or move this to be after the enroll cli users section.

Reply:
Good call. Moved the section to after the enroll CLI users and specifically called out the use of the Storm CLI command :+1: